### PR TITLE
[RA-1 Ch03] Change NFVI to cloud infrastructure

### LINKED ISSUE #1403: [RA01 Ch03] Change NFVI to Cloud Infrastructure
@pgoyal01 @rabi-abdel , Change the title of  Ch03 from "NFVI + VIM Architecture" to 
" Cloud Infrastructure Architecture - OpenStack " 

and add into the introduction, 
This CNTT Reference Architecture (RA-1) aims to provide an OpenStack distribution agnostic reference architecture which include NFVi layer ( VIM + NFVi )

### PR DESCRIPTION
**Pull request by pgoyal01**
Changes made; in some places instead of replacing NFVI, the sentence itself was changed.

Fixes #1403

### CODE REVIEW

**Comment by pgoyal01** [comment]
@ASawwaf Thanks -- Corrected; all occurrences changed to "Cloud Infrastructure." 

If in camel case then it would be "cloud Infrastructure"; I have used Pascal case.

File: doc/ref_arch/openstack/chapters/chapter03.md [hunk 1]
```diff
@@ -26,13 +26,13 @@
...
+This CNTT Reference Architecture (RA-1) aims to provide an OpenStack distribution agnostic reference architecture that includes the Network FUnction Virtualisation Infrastrcuture (NFVI) and Virtual Infrastructure Manager (VIM). The different OpenStack distributions, without the not up-streamed vendor specific enhancements, are assumed to be CNTT conformant. This Reference Architecture allows operators to provide a common OpenStack-based architecture for any CNTT compliant VNF to be deployed and operated as expected.  The purpose of this chapter is to outline all the components required to provide the Cloud Infrastructure (NFVI and the VIM) in a consistent and reliable way.
```
[the remark this answers is not in the document]
Comment (karinesevilla) [reply doc/ref_arch/openstack/chapters/chapter03.md]: Network FUnction Virtualisation Infrastrcuture
Typo: Infrastrcuture

[the remark this answers is not in the document]
Comment (pgoyal01) [reply doc/ref_arch/openstack/chapters/chapter03.md:29]: @ulikleber @karinesevilla Corrected - Thanks

**Comment by pgoyal01** [comment]
@karinesevilla Corrected (Ccloud) -- Thanks

**Pull request merged**